include/zephyr/bluetooth/mesh/blob_cli.h

```diff
@@ -419,6 +425,22 @@ uint8_t bt_mesh_blob_cli_xfer_progress_active_get(struct bt_mesh_blob_cli *cli);
  */
 bool bt_mesh_blob_cli_is_busy(struct bt_mesh_blob_cli *cli);
 
+/** @brief Set chunk sending interval in ms
+ *
+ *  This function is optional, and can be used to set how fast chunks get send in the BLOB Client
+ *  Model.
+ *  Without an added delay, for example a Mesh DFU can cause network blockage by constantly sending
+ *  the next chunks, especially if the chunks are send to group addresses or multiple unicast
+ *  addresses.
+ *
+ *  @note: Increasing the interval is not recommended for pull mode and can cause timeouts.
+ *  In labatory conditions an @p interval_ms above 500 ms with pull mode caused timeouts.
```

Comment:
I'm a bit uncertain if we should add this sentence. What configuration was used in the test, especially which `timeout_base` value? I understand how this can affect the pull mode, but I though that big interval between chunks can always be compensated by huge `timeout_base` value.

The opposite is not guaranteed either. Even with 100ms interval timeouts can happen because of the `timeout_base` value.

I think we should rather say that big interval may cause timeouts, therefore increase `timeout_base` value.